src/app/src/pages/Admin.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react"
+import { Box, List, ButtonGroup, Button , Badge, Typography, Divider} from "@mui/material"
+import NavBar from "../components/NavBar"
+import AdminRequestCard from "../components/AdminRequestCard"
+import AdminAuthorCard from "../components/AdminAuthorCard"
+import AdminPostCard from "../components/AdminPostCard"
+import AdminNodeCard from "../components/AdminNodeCard"
+
+export default function Admin(): JSX.Element {
+    const [listDisplay, setListDisplay] = React.useState({title:'Requests',id:0});
+
+    //Some fake data to help with layouts
+    const signupRequests = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        },
+    ];
+
+    const authors = [
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        displayName:"Lara Croft",
+        profileImage:null,
+        },
+        {
+        id:"c3293ed4-e55e-4986-8311-5ad43a27f5a3",
+        displayName:"Nathan Drake",
+        profileImage:null,
+        },
+    ];
+
+    const posts=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        author:authors[0],
+        date: "2022-02-25"
+        }
+    ];
+
+    const nodes=[
+        {
+        id:"07a931d8-b181-473d-8838-22dfb5c81416",
+        username:"NodeOne",
+        }
+    ];
+
+    // Get length for badges
+    const totalRequests = signupRequests.length;
+    const totalAuthors = authors.length;
+    const totalPosts = posts.length;
+    const totalNodes = nodes.length;
+
+    // Sidebar Button group
+    const buttons = [
+        <Button onClick={()=>setListDisplay({title:'Requests',id:0})}key="requests" sx={{justifyContent:"space-between", display: "flex"}}> Requests <Badge badgeContent={totalRequests} color="secondary" sx={{justifyContent:"right", mx:3}}/></Button>,
```

Comment:
@vyomea same changes here as well

Reply:
I merged from @osmani2 PR. She has a PR open just for the Admin page. I can do it for both files, I guess

Reply:
done

Reply:
Sorry, I missed this. I already handled seth's comments on admin in my own pr a while ago!

Reply:
I will change my file

Reply:
Please don't do this again, it's very confusing. I updated the tests so the PR can be merged but you should hold off, there are a few more things that need to be changed.